### PR TITLE
Week6BubbleSort

### PR DESCRIPTION
**Pull request by Giga-Chubinidze**

### CODE REVIEW

File: Bubbkesort_week6/lib/bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    sort
+  rescue InputOutputError => e
+    e.message
+  end
+
+  private
+
+  def sort
+    (0...arr.length).each do
+      (1..(arr.length - 1)).each do |idx|
```
Comment (alexshvedko) [inline Bubbkesort_week6/lib/bubble_sort.rb]: very deep. move inside loop to a new method

File: Bubbkesort_week6/spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```
Comment (alexshvedko) [inline Bubbkesort_week6/spec/spec_helper.rb]: remove all commented line
```
# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
# The generated `.rspec` file contains `--require spec_helper` which will cause
```

File: Bubbkesort_week6/lib/bubble_sort.rb (hunk 1 above)
Comment (Giga-Chubinidze) [reply Bubbkesort_week6/lib/bubble_sort.rb]: fixed it

File: Bubbkesort_week6/spec/spec_helper.rb (hunk 2 above)
Comment (Giga-Chubinidze) [reply Bubbkesort_week6/spec/spec_helper.rb]: fixed it

**Review by alexshvedko — APPROVED** [review]

**Pull request merged**